src/index.tsx

```diff
@@ -14,7 +13,7 @@ import {
     ViewProps,
 } from 'react-native'
 
-const FastImageViewNativeModule = NativeModules.FastImageView
```

Comment:
This line should remain because it is needed to use "clearMemoryCache" and "clearDiskCache". It also generate a typescript error.